### ISSUE TITLE
DHCP server broken on v0.105.0/snap 1687

### ISSUE THREAD

**Issue by marxjohnson** [AdguardTeam/AdGuardHome#2667]
### Issue Details

* **Version of AdGuard Home server:**
  * v0.105.0
* **How did you install AdGuard Home:**
  * snap
* **How did you setup DNS configuration:**
  * System
* **If it's a router or IoT, please write device model:**
  * 
* **Operating system and version:**
  * Ubuntu 18.04.5

### Expected Behavior
DHCP server continues to work after the snap updated.

### Actual Behavior
DHCP server does not respond to requests. Disabling and re-enabling the DCHP server through the web interface results in the error:
`Error: control/dhcp/set_config | enabling dhcp: checking static ip: open /etc/network/interfaces: permission denied | 500`

### Additional Information
Previous issues with the DHCP server were resolved by running `sudo snap connect adguard-home:network-control`. This now results in the message `error: snap "adguard-home" has no plug named "network-control"`.

Running `sudo snap revert adguard-home` reverted to v0.104.3, which works as before.

Other information about the snap:
```
$ snap info adguard-home
name:      adguard-home
summary:   Network-wide ads & trackers blocking DNS server
publisher: AdGuard (ameshkov✓)
store-url: https://snapcraft.io/adguard-home
  thus preventing your devices from connecting to those servers. It's based
  on software we use for our public AdGuard DNS servers -- both share a lot
  of common code.
  
  IMPORTANT:
  1. Once it's installed, open http://127.0.0.1:3000/ in your browser to do the initial setup and
  learn how to configure your devices to use AdGuard Home. You don't need to Launch anything
commands:
  - adguard-home.adguard-home-web
services:
  adguard-home: simple, enabled, active
snap-id:      UXZIkJfJT2SPCGejjnSjOBqJ71yHk8bw
store-url: https://snapcraft.io/adguard-home
  you set it up, it'll cover ALL your home devices, and you don't need any
  client-side software for that.
  
  It operates as a DNS server that re-routes tracking domains to a "black hole,"
  thus preventing your devices from connecting to those servers. It's based
  on software we use for our public AdGuard DNS servers -- both share a lot
  of common code.
  
  IMPORTANT:
  1. Once it's installed, open http://127.0.0.1:3000/ in your browser to do the initial setup and
  learn how to configure your devices to use AdGuard Home. You don't need to Launch anything
  explicitly, AdGuard Home service is already started after the installation.
  2. If you're running it on Ubuntu, you may need to select a different "Listen interface" for the
  DNS server. Listening to all interfaces is not possible because Ubuntu's systemd-resolved service
  uses 127.0.0.1:53 already.
commands:
  - adguard-home.adguard-home-web
services:
  adguard-home: simple, enabled, active
snap-id:      UXZIkJfJT2SPCGejjnSjOBqJ71yHk8bw
tracking:     latest/stable
refresh-date: yesterday at 22:38 UTC
channels:
  latest/stable:    v0.105.0        2021-02-10 (1687) 10MB -
  latest/candidate: ↑                                      
  latest/beta:      v0.105.0-beta.5 2021-02-10 (1682) 10MB -
  latest/edge:      v0.105.0        2021-02-10 (1687) 10MB -
installed:          v0.105.0                   (1687) 10MB -
```
```
$ snap connections adguard-home 
Interface        Plug                          Slot           Notes
desktop          adguard-home:desktop          :desktop       -
network-bind     adguard-home:network-bind     :network-bind  -
network-observe  adguard-home:network-observe  -              -
```

**Comment by ainar-g** [comment]
Hi, and thank you for the report!  Does `sudo snap connect adguard-home:network-observe` fix the issue?  We've been advised by the people working in Snap Store to not use `network-control`, as it may provide too much access to the host machine, but `network-observe` should work for most cases.

**Comment by marxjohnson** [comment]
I tried as you suggested, but to no avail:
```
mark@strongmad:~$ sudo snap refresh adguard-home 
adguard-home v0.105.0 from AdGuard (ameshkov✓) refreshed
mark@strongmad:~$ sudo snap connect adguard-home:network-observe
mark@strongmad:~$ snap connections adguard-home 
Interface        Plug                          Slot              Notes
desktop          adguard-home:desktop          :desktop          -
network-bind     adguard-home:network-bind     :network-bind     -
network-observe  adguard-home:network-observe  :network-observe  manual
```
After this, the error occurred as before.

**Comment by ainar-g** [comment]
We've pushed some fixes in snapshot 10f03b75.  It should already be on the edge channel in the Snap store.  Can you please check if our solution fixes the issue for you?

**Comment by marxjohnson** [comment]
I can confirm that snapshot fixes the issue.
